extension/src/config/constants.js

```diff
@@ -19,5 +19,8 @@ module.exports = {
   CTP_CUSTOM_FIELD_GET_ORIGIN_KEYS_RESPONSE: 'getOriginKeysResponse',
   CTP_INTERACTION_TYPE_MAKE_PAYMENT: 'makePayment',
   CTP_CUSTOM_FIELD_MAKE_PAYMENT_REQUEST: 'makePaymentRequest',
-  CTP_CUSTOM_FIELD_MAKE_PAYMENT_RESPONSE: 'makePaymentResponse'
+  CTP_CUSTOM_FIELD_MAKE_PAYMENT_RESPONSE: 'makePaymentResponse',
+  CTP_INTERACTION_TYPE_SUBMIT_PAYMENT_DETAILS: 'submitAdditionalPaymentDetails',
+  CTP_CUSTOM_FIELD_SUBMIT_PAYMENT_DETAILS_REQUEST: 'submitAdditionalPaymentDetailsRequest',
```

Comment:
Those things not important, and you could skip that comment, but it would be nice if you check again, the string "submitPaymentDetails" and replace them with "submitAdditionalPaymentDetails" for consistency. There are lots of   :)

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/271/commits/547a3f2a094f121ea1a077d24abca5ddef55d961